astropy_helpers/sphinx/ext/doctest.py

```diff
@@ -37,6 +45,8 @@ def setup(app):
     app.add_directive('doctest-requires', DoctestRequiresDirective)
     app.add_directive('doctest-skip', DoctestSkipDirective)
     app.add_directive('doctest-skip-all', DoctestSkipDirective)
+    app.add_directive('doctest', DoctestSkipDirective)
+    app.add_directive('testsetup', DoctestOmitDirective)
```

Comment:
Could you add a comment here explaining how ``DoctestSkipDirective`` is different from ``DoctestOmitDirective``?